### DISCUSSION TITLE
Automatic budgets not updated for new month

### DISCUSSION THREAD

**Discussion by GabrielBarcia** [firefly-iii/firefly-iii#5147] [Firefly III] [answered]
Hi, today I see that the automatic budgets are not populated in the new month (october), in previous months this was not a problem.
![image](https://user-images.githubusercontent.com/4810944/135595190-9c0beab6-5c17-459c-b038-457aba558ddb.png)

I suspect that it may have something to do with the fact that the server was shutdown maybe yesterday until today morning?
Any advise how to fix it in any other way that is not manually?

thanks
Regards
Gabriel

**Comment by JC5** [comment, marked as answer]
Yes that's a possibility, if your cron job would have run during that time.

**Reply by GabrielBarcia** [reply]
Yes, the cron job would have run during the time it was shutdown. Should it update in the next chron run tomorrow?

**Reply by JC5** [reply]
No, it will only create them on the first of the period, ie. on the first.

Run it yourself like this:

`php artisan firefly-iii:cron --force --date=2021-10-01`

**Reply by GabrielBarcia** [reply]
Yes, this works. Just need to run with the www-data user due to folder ownership

`sudo runuser -u  www-data -- php artisan firefly-iii:cron --force --date=2021-10-01`

Thanks
